script/1_0_2current_en_script/Teach_Me_Ciel-Sensei/QA_04.txt

```diff
@@ -11,7 +11,7 @@ Pardon me. The big battle had finally begun,%{n}
 so I got a bit carried away...
 <Page5>
 This Q&A corner is brought to you by your guiding
```

Comment:
This Q&A Corner is brought to you by your host:
me, Ciel together with this outsider cat.

Also, why space added at the start?

Reply:
Hacky line break things :) Please never remove extra spaces at the beginning of lines in QA files.